asn1/InterledgerProtocol.asn

```diff
@@ -4,76 +4,52 @@ AUTOMATIC TAGS ::=
 BEGIN
 
 IMPORTS
-    UInt64
+    UInt64,
+    UInt256
     FROM GenericTypes
 
     Address,
     Timestamp
     FROM InterledgerTypes
 ;
 
--- the following packet type is experimental and may still change
-ForwardedPaymentData ::= SEQUENCE {
-    -- Destination ILP Address
-    account Address,
-    -- Information for recipient (transport layer information)
-    data OCTET STRING (SIZE (0..32767)),
-    -- Enable ASN.1 Extensibility
-    extensions SEQUENCE {
-        ...
-    }
-}
-
-InterledgerProtocolPaymentData ::= SEQUENCE {
-    -- Amount which must be received at the destination
+InterledgerPrepare ::= SEQUENCE {
+    -- Local amount (changes at each hop)
     amount UInt64,
```

Comment:
Why is the amount back in the ILP packet?

Reply:
This time it's the transfer amount, not the destination amount

Reply:
I don't follow why this is required at this layer?

Reply:
Because of https://github.com/interledger/rfcs/issues/359